src/app/scripts/cac/map/cac-map-isochrone.js

```diff
@@ -96,12 +96,12 @@ CAC.Map.IsochroneControl = (function ($, Handlebars, cartodb, L, turf, _, Settin
     }
 
     /**
-     * Fetch an isochrone (travelshed).
+     * Fetch an isochrone (travelshed) and the places within it.
      *
-     * @return {Object} Promise resolving to JSON with isochrone
+     * @return {Object} Promise resolving to JSON with isochrone and destinations
      */
     function _fetchIsochrone(payload) {
-        var isochroneUrl = Settings.isochroneUrl;
+        var isochroneUrl = '/map/reachable';
```

Comment:
The `isochroneUrl` was made configurable because the OTP server lcoation depends on the deployment context. Since this is referencing the same Django server that serves this front-end app, it's not necessary to parameterize it; it can be a relative URL.